### ISSUE TITLE
Midterm Peer Review

### ISSUE THREAD

**Issue by jiangckevin** [kathy908000/Population-Segmentation-For-Health-System-Management#9]
The project seeks to predict the length of stay and total cost of cancer patients in New York state hospitals using demographic and patient medical information. The data set, as implied by the title of the project, is New York State's "Statewide Planning and Research Cooperative System's (SPARCS) Hospital Inpatient Discharges." 

What I liked:

(i) The preprocessing is clearly explained and well thought out. Ordinal features (e.g Risk of Mortality), categorical features, and binary features are listed and preprocessed with accordingly.

(ii) The figures are spot on. They are quite clean and easy to interpret. I especially liked the Cross-validation statistics table as it allowed for easy viewing on which features were least likely to overfit with your introduced bootstrap method.

(iii) I liked how you explained and walked through the methodology for each one. For instance, the leave one feature out cross validation-esque method for linear regression and the specification of hyperparameters for the random forest allowed for easy-to-follow reading. I appreciated the effort and though given to this.

Some critique:

(i) I would recommend introducing a loss function that puts a heavier penalty on false negatives than false positives, as their difference in the real world is of great importance. 

(ii) The "manual screening" of the Linear Regression seems a bit sketchy, in that you purposefully excluded issues that seemed convenient for the model at hand. Perhaps a more rigorous explanation could be given? (e.g. getting rid of features based on collinearity makes sense, and you could reason why such a process is not "manual" but rather a necessary part of preprocessing)

(iii) In the Random Forest section, you list hyperparameters in the text, and this is a little confusing (at least to me). Perhaps a figure like the one with Cross-validation Statistics would make it more appealing if not more clear?
